erpnext/public/js/utils.js

```diff
@@ -37,6 +37,10 @@ $.extend(erpnext, {
 		}
 	},
 
+	allow_stale_rate: () => {
```

Comment:
Let's name it `stale_rate_allowed`. `allow_stale_rate` can lead readers to think allow_stale_rate is being enabled